common/persistence/sql/sqlplugin/postgresql/plugin.go

```diff
@@ -41,8 +41,8 @@ import (
 
 const (
 	// PluginName is the name of the plugin
-	PluginName    = "postgres"
-	PluginNamePGX = "postgres_pgx"
+	PluginNameV12    = "postgres12"
```

Comment:
what is the reason we have to change constant names, not just values?

Reply:
I'll follow up in another PR to do this. I didn't want to pollute too much this PR with renaming changes.